### ISSUE TITLE
Fix "Update Lock" when date_picked == unlock_date

### ISSUE THREAD

**Issue by idiom-bytes** [oceanprotocol/df-web#398]
### Problem:  
**tldr; if date_compare == same day, button should remain disabled.**  
You can't just update your lock, and you'll get an error. So disable it.   
The only way you can do that, is by changing the amount.  
  
From the work done in #397 (https://github.com/oceanprotocol/df-web/pull/397/files)

### DoD:
- [ ] When the user selects the same date, as the existing unlock date, the button should not be enabled. Only if they have a higher amount, as the logic is already written.

I think I used moment to output some conditional checks (didn't output what I expected, might need some tweaking, or could be written more easily w/ >< conditions.
```
  $: if ($form) {
    updateLockButtonText = getUpdateLockButtonText();

    console.log("$form.amount", $form.amount)
    console.log("$form.unlockDate", $form.unlockDate)
    console.log("$oceanUnlockDate", $oceanUnlockDate)
    if($oceanUnlockDate) {
      console.log("$oceanUnlockDate.utc", $oceanUnlockDate.utc())
      console.log("$oceanUnlockDate.utc.format()", $oceanUnlockDate.utc().format("YYYY-MM-DD"))
      console.log("isSame simple compare:", $oceanUnlockDate.utc().format("YYYY-MM-DD") === $form.unlockDate)
      console.log("isBefore:", moment($form.unlockDate).isBefore($oceanUnlockDate.utc().format()))
      console.log("isSame:", moment($form.unlockDate).isSame($oceanUnlockDate.utc().format()))
      console.log("isSameOrBefore:", moment($form.unlockDate).isSameOrBefore($oceanUnlockDate.utc().format()))
      console.log("isAfter:", moment($form.unlockDate).isAfter($oceanUnlockDate.utc().format()))
      console.log("isSameOrAfter:", moment($form.unlockDate).isSameOrAfter($oceanUnlockDate.utc().format()))
    }
  }
```
